docs/source/using/troubleshooting.adoc

```diff
@@ -11,6 +11,26 @@ toc::[]
 
 This section contains solutions to common problems that you might encounter while using Minishift.
 
+[[minshift-update-failed-due-to-permission-denied]]
+== Permission cause update to fail
+
+Depending on your operating system if you have minishift binary installed in location where current user does not have write permission then `minishift update` will fail.
+
```

Comment:
@hferentschik If the directory is not writable at all then it is a different issue isn't it? Otherwise how the minishift binary is present in the location.